Source/Core/Core/Core.cpp

```diff
@@ -806,6 +801,17 @@ static bool PauseAndLock(bool do_lock, bool unpause_on_unlock)
   return was_unpaused;
 }
 
+static void ResetRumble()
+{
+#if defined(__LIBUSB__)
+  GCAdapter::ResetRumble();
+#endif
+#if defined(CIFACE_USE_XINPUT) || defined(CIFACE_USE_DINPUT)
```

Comment:
You can use `#elif` instead of using `#endif` followed by `#if`.

Reply:
I don't think the two cases should exclude each other though, otherwise GC adapters won't be getting reset on Windows anymore.

Reply:
Oh, sorry. Yes, ignore my comment.